### ISSUE TITLE
Inaccessible checkboxes

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3120]
Several checkboxes are used in the hierarchical structure of the "Scene" menu. The checkboxes are implemented as icon elements with a text alternative of "check_box".  With the current implementation, the custom controls cannot be focused by keyboard users, while the visual affordances that indicate the name, role and state of a checkbox are not conveyed to screen reader users. 

**Suggested Solution** 
Implement checkboxes as native `<input>` elements with type="checkbox" and  associate the visible label with the checkbox using the for attribute on the `<label>`  referencing the ID of the `<input>`. 

Severity: 1

**Comment by engbergandreas** [comment]
Closed with https://github.com/OpenSpace/OpenSpace/commit/e3f5b83f7476c8507dccd45ab98e4731504af532
